src/Definition/Doc.elm

```diff
@@ -560,32 +610,42 @@ view refToMsg toggleFoldMsg docFoldToggles document =
 -- DECODE
 
 
-decodeSpecialForm : Decode.Decoder SpecialForm
-decodeSpecialForm =
+decodeSpecialForm : TreePath -> Decode.Decoder SpecialForm
+decodeSpecialForm path =
```

Comment:
This function mainly changed to look for code in the right JSON fields and to add the TreePath collection.